src/screens/ForgotPassword/index.tsx

```diff
@@ -12,10 +12,9 @@ const ForgotPassword = (): JSX.Element => {
   const { requestSuccess, error, handleSubmit } = SubmitHandler()
 
   return (
-    <Background backgroundImage={authBackground}>
+    <Background backgroundImage={authBackground} pageClassName="forgot-password">
       <BackButton />
-      {/* TODO: DRY the container styling with Sign in page */}
-      <div className="container-forgot-password">
+      <div className="container-auth">
```

Comment:
I feel like screens should have different names. It would become a problem if you were to change something again for a particular screen as it doesn't concern the other screens. How about using CSS mixins if you want to DRY this styles? Let me know your thoughts.

Reply:
I have a page name on Background component already. Is it ok for you?
<img width="881" alt="Screen Shot 2564-06-29 at 14 36 30" src="https://user-images.githubusercontent.com/29707647/123756708-66486e80-d8e7-11eb-90f0-c4d3212e7bb4.png">

Reply:
That sounds nice. 👍